src/components/FilterDivider.tsx

```diff
@@ -0,0 +1,12 @@
+import { twMerge } from '../hooks/useComposedCssClasses';
+/**
+ * A divider component for NumericalFacets, HierarchicalFacets, and StandardFacets.
```

Comment:
can you also add that this divider should be used in between `StaticFilters` components to separate them?